lib/fuzzy-finder-view.js

```diff
@@ -390,27 +390,32 @@ module.exports = class FuzzyFinderView {
     return {uri: filePath, filePath, label}
   }
 
-  getAbsolutePath (relativePath) {
-    const directories = atom.project.getDirectories()
+  parseResultsFromFuzzyNative (results) {
```

Comment:
This is minor, but I think "parse" isn't really the right word here. Maybe `convertFuzzyNativeResultsToViews` or something along those lines?